mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -389,6 +436,17 @@ class ALSModel private[ml] (
     )
     recs.select($"id" as srcOutputColumn, $"recommendations" cast arrayType)
```

Comment:
This is discouraged within Spark: https://github.com/databricks/scala-style-guide#infix-methods